assets/css/navbar-responsive.css

```
@@ -0,0 +1,1313 @@
+  }
+
+  .site-header #nav.navbar.is-fresh .navbar-menu a.navbar-item:hover svg.icon,
+  .site-header #nav.navbar.is-fresh .navbar-menu a.navbar-item:focus-visible svg.icon,
+  .site-header #nav.navbar.is-fresh .navbar-menu .navbar-link:hover svg.icon,
+  .site-header #nav.navbar.is-fresh .navbar-menu .navbar-link:focus-visible svg.icon,
+  #navbar-clone.navbar.is-fresh .navbar-menu a.navbar-item:hover svg.icon,
+  #navbar-clone.navbar.is-fresh .navbar-menu a.navbar-item:focus-visible svg.icon,
+  #navbar-clone.navbar.is-fresh .navbar-menu .navbar-link:hover svg.icon,
+  #navbar-clone.navbar.is-fresh .navbar-menu .navbar-link:focus-visible svg.icon {
+    color: var(--openms-navbar-link-color);
+    fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)